src/client/Microsoft.Identity.Client/OAuth2/MsalTokenResponse.cs

```diff
@@ -73,10 +76,13 @@ public MsalTokenResponse()
         [JsonExtensionData]
         public Dictionary<string, JToken> ExtensionData { get; set; }
 #endif
-
+#endif
         // Exposes only scalar properties from ExtensionData
         public Dictionary<string, string> CreateExtensionDataStringMap()
         {
+#if __MOBILE__
+            return null;
+#else
             if (ExtensionData == null || ExtensionData.Count == 0)
             {
                 return null;

```

Comment:
nit (not for this PR) - we should return empty collections rather than nulls, to reduce chance of NPEs.